src/components/data-table/data-column.ts

```diff
@@ -10,7 +10,7 @@ export interface DataColumn<T, F extends DataTableFilterItem = DataTableFilterIt
     name: string;
     selected: boolean;
     configurable: boolean;
-    sortDirection: SortDirection;
+    sortDirection?: SortDirection;
```

Comment:
why `?` ?

Reply:
Sort direction is meant to control sort direction and whether a column can be sorted